### ISSUE TITLE
Syntax error when compiling on macOS.

### ISSUE THREAD

**Issue by huyphams** [snowflakedb/libsnowflakeclient#107]
I got this error when compiling the lasted release (0.2.5) of libsnowflakeclient on macOS 10.11

```
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:59:33: error: use of undeclared identifier 'strtoul'
        m_port = (unsigned int) strtoul(proxy.substr(pos, (proxy.length()) - pos).c_str(), nullptr, 10);
                                ^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:74:14: error: no member named 'getenv' in namespace 'std'
    if (std::getenv("all_proxy")) {
        ~~~~~^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:75:22: error: no member named 'getenv' in namespace 'std'
        proxy = std::getenv("all_proxy");
                ~~~~~^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:76:21: error: no member named 'getenv' in namespace 'std'
    } else if (std::getenv("https_proxy")) {
               ~~~~~^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:77:22: error: no member named 'getenv' in namespace 'std'
        proxy = std::getenv("https_proxy");
                ~~~~~^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:78:21: error: no member named 'getenv' in namespace 'std'
    } else if (std::getenv("http_proxy")) {
               ~~~~~^
/Users/huy/Desktop/libsnowflakeclient-0.2.5/cpp/util/Proxy.cpp:79:22: error: no member named 'getenv' in namespace 'std'
        proxy = std::getenv("http_proxy");
```

It is fixed if I added this line in the `Proxy.hpp`
```
 #include <cstdlib>
```

And one minor issue that the lib says its version is 0.2.3 but your tag release is 0.2.5

**Comment by notkriswagner** [comment]
Thanks for pointing this out. I've fixed both of these in the latest release, v0.2.6. Let me know if this fixes the issues on your end.

**Comment by huyphams** [comment]
I confirm that it was fixed.
